src/Sylius/Bundle/AdminBundle/Resources/views/Shipment/Component/ship.html.twig

```diff
@@ -0,0 +1,16 @@
+{# Rendered with using \Sylius\Bundle\AdminBundle\TwigComponent\Shipment\ShipType #}
+
+{% form_theme form '@SyliusAdmin/Shared/form_theme.html.twig' %}
+
+{% if sm_can(shipment, 'ship', 'sylius_shipment') %}
```

Comment:
Leaving a comment to not forget about it, we need our state machine abstraction twig function equivalent for this.